### PR TITLE
Load srdf in upload_robot.launch

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
As discussed https://github.com/ipa320/cob_robots/pull/468

This pull request includes https://github.com/ipa320/cob_robots/pull/479

**Comment by NadiaHG** [comment]
Test on cob4-1: OK

File: cob_bringup/tools/collision_monitor.launch [hunk 1]
```diff
@@ -2,17 +2,6 @@
 <launch>
 
 	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
```
Comment (mathias-luedtke) [inline cob_bringup/tools/collision_monitor.launch]: is this line kept by policy?

Comment (NadiaHG) [inline cob_bringup/tools/collision_monitor.launch]: No, actually is not needed any more, I will remove the line.

**Comment by floweisshardt** [comment]
CR: OK

**Comment by NadiaHG** [comment]
Travis is now happy!

**Pull request merged**